src/patternfly/components/FileUpload/file-upload.scss

```diff
@@ -34,6 +34,7 @@
 
   &.pf-m-drag-hover {
     position: relative;
+    border: var(--pf-c-file-upload--m-drag-hover--BorderWidth) solid var(--pf-c-file-upload--m-drag-hover--BorderColor);
```

Comment:
adding a border is going to make the component shrink/grow when the class is applied. You could use the `::before` pseudo class to draw the border